Server/Core/Commands/TCPConnectionsHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Threading;
+using xServer.Core.Networking;
+using xServer.Core.Packets.ClientPackets;
+
+namespace xServer.Core.Commands
+{
+    /* THIS PARTIAL CLASS SHOULD CONTAIN METHODS THAT HANDLE TCP Connections COMMANDS. */
+    public static partial class CommandHandler
+    {
+        public static void HandleGetConnectionsResponse(Client client, GetConnectionsResponse packet)
+        {
+
+            if (client.Value == null || client.Value.FrmCon == null)
+                return;
+
+            client.Value.FrmCon.ClearListviewItems();
+
+            // None of the arrays containing the process' information can be null.
+            // The must also be the exact same length because each entry in the five
+            // different arrays represents one process.
+            if (packet.Processes == null || packet.LocalAddresses == null || packet.LocalPorts == null ||
+                packet.RemoteAddresses == null || packet.RemotePorts == null || packet.States == null ||
+                packet.Processes.Length != packet.LocalAddresses.Length || packet.Processes.Length != packet.LocalPorts.Length ||
+                packet.Processes.Length != packet.RemoteAddresses.Length || packet.Processes.Length != packet.RemotePorts.Length ||
+                packet.Processes.Length != packet.States.Length)
+
+                return;
+
+            new Thread(() =>
```

Comment:
Just out of curiosity; how long would this thread be working? Is it really worth creating a new thread each time (especially since we do most of the work on the <code>client.Value.FrmCon.AddConnectionToListview</code> method)?